src/ca/idcert.rs

```diff
@@ -178,23 +182,30 @@ impl IdCert {
     /// by the provided `issuer` certificate.
     ///
     /// Note that this does _not_ check the CRL.
-    pub fn validate_ee(&self, issuer_key: &PublicKey) -> Result<(), ValidationError> {
+    pub fn validate_ee(
+        &self, issuer_key: &PublicKey,
+    ) -> Result<(), ValidationError> {
         self.validate_ee_at(issuer_key, Time::now())
     }
 
-    pub fn validate_ee_at(&self, issuer_key: &PublicKey, now: Time) -> Result<(), ValidationError> {
-        self.validate_basics(now)?;
-        self.validate_aki(issuer_key)?;
+    pub fn validate_ee_at(
```

Comment:
What is the difference between validation and verification?

Reply:
This is explained in the doc string for `repository::cert::Cert`. We could copy this over – or maybe explain the steps in the module doc for `repository::error`.